webtau-config/src/main/java/com/twosigma/webtau/cfg/WebTauConfig.java

```diff
@@ -33,14 +33,16 @@
 import static com.twosigma.webtau.cfg.ConfigValue.declareBoolean;
 
 public class WebTauConfig {
-    public static String CONFIG_FILE_NAME_DEFAULT = "webtau.cfg";
+    public static final String REPL_KEY = "repl";
+    public static final String CONFIG_FILE_NAME_DEFAULT = "webtau.cfg";
 
     private static final List<WebTauConfigHandler> handlers = discoverConfigHandlers();
 
     private static final Supplier<Object> NO_DEFAULT = () -> null;
 
     private final ConfigValue config = declare("config", "config file path", () -> CONFIG_FILE_NAME_DEFAULT);
-    private final ConfigValue interactive = declareBoolean("interactive", "use CLI interactive mode");
+    private final ConfigValue interactive = declareBoolean("interactive", "use CLI interactive mode (experimental)");
+    private final ConfigValue repl = declareBoolean(REPL_KEY, "use CLI repl mode (will scuppered interactive)");
```

Comment:
now here's a word you don't see every day, `scupper`.
Did you mean to say "will replace interactive mode"?

Reply:
ops, how did this happen? I meant to type "supersede"